packages/@vue/cli-plugin-unit-mocha/index.js

```diff
@@ -37,9 +37,17 @@ module.exports = api => {
       `https://sysgears.github.io/mochapack/docs/installation/cli-usage.html`
     )
   }, (args, rawArgv) => {
-    const inspectPos = rawArgv.indexOf('--inspect-brk')
```

Comment:
I think the functionality can be achieved with a one-liner:

```js
const inspectPos = rawArgv.findIndex(arg => arg.startsWith('--inspect-brk'))
```

Reply:
Yes indeed - that is a cleaner implementation. I have pushed a change to reflect your suggestion.

Reply:
Thanks!